test/test_cuda.py

```diff
@@ -3912,8 +3912,11 @@ def test_cachingAllocator_raw_alloc(self):
         # relevant field in data structure
         def requested_bytes_alloc_stats(raw_alloc_size, stream):
             start = torch.cuda.memory_stats()["requested_bytes.all.allocated"]
-            torch._C._cuda_cudaCachingAllocator_raw_alloc(raw_alloc_size, stream)
+            mem_ptr = torch._C._cuda_cudaCachingAllocator_raw_alloc(
+                raw_alloc_size, stream
+            )
             finish = torch.cuda.memory_stats()["requested_bytes.all.allocated"]
+            torch.cuda.caching_allocator_delete(mem_ptr)
```

Comment:
```suggestion
            torch._C._cuda_cudaCachingAllocator_raw_delete(mem_ptr)
```
nit: for symmetry with the raw_alloc above even though torch.cuda.caching_allocator_delete() immediately calls raw_delete.